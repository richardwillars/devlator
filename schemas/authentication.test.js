const authenticationSchemas = require("./authentication");

describe("schemas/authentication", () => {
  it("should export the schemas", () => {
    expect(authenticationSchemas).toMatchSnapshot();
  });
});
