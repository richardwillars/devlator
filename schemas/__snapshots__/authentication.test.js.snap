// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`schemas/authentication should export the schemas 1`] = `
Object {
  "requested": Object {
    "ManualAction": Object {
      "properties": Object {
        "message": Object {
          "type": "string",
        },
        "type": Object {
          "type": "string",
        },
      },
      "required": Array [
        "type",
        "message",
      ],
      "type": "object",
    },
    "RequestData": Object {
      "properties": Object {
        "button": Object {
          "properties": Object {
            "label": Object {
              "type": "string",
            },
            "url": Object {
              "type": "string",
            },
          },
          "required": Array [
            "url",
            "label",
          ],
          "type": "object",
        },
        "dataLabel": Object {
          "type": "string",
        },
        "message": Object {
          "type": "string",
        },
        "type": Object {
          "type": "string",
        },
      },
      "required": Array [
        "type",
        "message",
        "dataLabel",
      ],
      "type": "object",
    },
  },
  "returned": Object {
    "ManualAction": Object {
      "properties": Object {
        "message": Object {
          "type": "string",
        },
        "success": Object {
          "type": "boolean",
        },
      },
      "required": Array [
        "success",
      ],
      "type": "object",
    },
    "RequestData": Object {
      "properties": Object {
        "data": Object {
          "type": "string",
        },
      },
      "required": Array [
        "data",
      ],
      "type": "object",
    },
  },
}
`;
